avaframe/com1DFA/com1DFA.py

```diff
@@ -253,14 +256,14 @@ def prepareRelase(cfg, rel, inputSimLines):
         log.warning('Release area scenario file name includes an underscore \
         the suffix _AF will be added for the simulation name')
     releaseLine = inputSimLines['releaseLine']
-    releaseLine['relThSource'] = [''] * len(releaseLine['relTh'])
-    for k in range(len(releaseLine['relTh'])):
-        if cfg['GENERAL'].getboolean('useRelThFromIni') or releaseLine['relTh'][k] == 'None':
-            releaseLine['relTh'][k] = cfg['GENERAL'].getfloat('relTh')
-            releaseLine['relThSource'][k] = 'ini file'
+    releaseLine['thicknessSource'] = [''] * len(releaseLine['thickness'])
+    for k in range(len(releaseLine['thickness'])):
+        if cfg['GENERAL'].getboolean('useRelThFromIni') or releaseLine['thickness'][k] == 'None':
+            releaseLine['thickness'][k] = cfg['GENERAL'].getfloat('relTh')
+            releaseLine['thicknessSource'][k] = 'ini file'
         else:
-            releaseLine['relTh'][k] = float(releaseLine['relTh'][k])
-            releaseLine['relThSource'][k] = 'shp file'
+            releaseLine['thickness'][k] = float(releaseLine['thickness'][k])
+            releaseLine['thicknessSource'][k] = 'shp file'
```

Comment:
This (and the other similiar ones) can be simplified (at least I think so):
- if use from ini is set, all release lines get the same value, ie. all thicknesses and source can be set at once
- if shp file is set, all sources can be set at once,
- therefore the only loop that is needed is that for the conversion to float